.asf.yaml

```diff
@@ -30,7 +30,7 @@ github:
     rebase: true
   protected_branches:
     master: { }
-    maven-javadoc-plugin-3.8.3: { }
+    maven-javadoc-plugin-3.8.x: { }
```

Comment:
Not sure we really plan to make new releases from this branch 
Protecting this is not really needed. Branch could be even deleted